docs/configuration/git-providers/bitbucket-enterprise.md

```diff
@@ -4,12 +4,14 @@ Set your configuration values for your Bitbucket instance on the `values.yaml` f
 
 **NOTE:** Since Bitbucket Server uses OAuth1, you'll need to create a key pair to sign and validate the requests between Codacy and the Bitbucket Server instance:
 
-1. Create the key pair by running the command below, making sure that you don't define a passphrase:
+1. Create a key pair using the RSA algorithm in the PKCS#8 format by running the following commands below, making sure that you don't define a passphrase:
 
-   `ssh-keygen -t rsa -f mykey -q -N ""`
+   1. `openssl genrsa -out mykey`
+   1. `openssl pkcs8 -nocrypt -in mykey -out mykeypkcs8 -topk8`
+   1. `openssl rsa -in mykey -pubout -out mykey.pub`
```

Comment:
Consider using a code block instead for all three commands:

```
    ```bash
    openssl genrsa -out mykey
    openssl pkcs8 -nocrypt -in mykey -out mykeypkcs8 -topk8
    openssl rsa -in mykey -pubout -out mykey.pub
    ```
```